eventservice/api/controllers/events.py

```diff
@@ -350,6 +350,39 @@ def patch(event_id):
         __logger.exception(ex)
         abort(405)
 
+    group_memberships = list()
+    try:
+        _, group_memberships = get_group_memberships()
+    except Exception as ex:
+        __logger.exception(ex)
+        abort(500)
+
+    db = None
+    event = None
+    try:
+        db = get_db()
+        event = db['events'].find_one({'_id': ObjectId(event_id)}, {'_id': 0})
+    except Exception as ex:
+        __logger.exception(ex)
+        abort(500)
+
+    if not group_memberships:
```

Comment:
Hi @bingzhang, This code block needs to be updated based on the changes from Friday, right? Please look into this. Thanks.